src/deployCommand.ts

```diff
@@ -259,3 +271,26 @@ export const getCoverageFormattersOptions = (formatters: string[] = []): Coverag
     reportOptions,
   };
 };
+
+export const targetUsernameFlag = Flags.string({
+  required: true,
+  char: 'u',
+  deprecateAliases: true,
+  // DO NOT alias to 'o', it will conflict with '--ignoreerrors'
+  aliases: ['targetusername', 'u'],
+  summary: messages.getMessage('flags.targetusername.summary'),
+  parse: async (input: string | undefined) => resolveUsername(input),
+  default: async () => resolveUsername(),
+  defaultHelp: async () => resolveUsername(),
```

Comment:
Is this correct for defaultHelp?

Reply:
This is similar to what we did in plugin-org

https://github.com/salesforcecli/plugin-org/pull/646/files#diff-2cb023071675d0d3c17b86b021f81fd33622048a9eab2fb3d6623e52375f2690R38

Reply:
This form can leak customer data into the manifest.

Reply:
@peternhale ^